lms/djangoapps/ccx/views.py

```diff
@@ -233,10 +233,16 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
                 ccx_ids_to_delete.append(get_override_for_ccx(ccx, block, 'start_id'))
                 clear_ccx_field_info_from_ccx_map(ccx, block, 'start')
 
-            due = parse_date(unit['due'])
-            if due:
-                override_field_for_ccx(ccx, block, 'due', due)
+            # Only subsections (aka sequential) and units has due date.
```

Comment:
I think this clearer & more complete: 

`Only subsection (aka sequential) and unit (aka vertical) have due dates.`